pages/landing/enterprise-eks/index.html

```diff
@@ -0,0 +1,191 @@
+      accounts and VPCs, or you can dive right in to the code. Or, use <a href="https://gruntwork.io/landing-zone-for-aws/">Gruntwork Landing Zone</a> and the
+      <a href="https://gruntwork.io/reference-architecture">Gruntwork Reference Architecture</a> for a complete,
+      end-to-end stack, built with extensibility in mind. No matter what you choose, everything is managed as code with Terraform.
+    logos:
+      - title: Terraform
+        src: /assets/img/landing-heroku-page/terraform-icon.png
+      - title: Docker
+        src: /assets/img/landing-heroku-page/docker-icon.png
+      - title: Packer
+        src: /assets/img/landing-heroku-page/packer-icon.png
+
+  - heading: "Step 3: Set up your app"
+    image: /assets/img/landing-page-enterprise/sample-app-clip.gif
+    image_side: right
+    body: |
+      Use Gruntwork tools to optimize your enterprise apps for EKS. Update your app with cloud-native best practices for
+      packaging, monitoring, logging, service discovery, secrets management, schema migrations, and more. Or customize
+      the optimizations yourself using Gruntwork sample apps as a template. We include sample apps common enterprise languages and
+      frameworks:
+      <ul>
+          <li>Spring boot</li>
+          <li>Standard Java apps (e.g., Tomcat or similar)</li>
+          <li>Ruby on Rails</li>
+          <li>Python Django</li>
+          <li>Or suggest something!</li>
+      </ul>
+    logos:
+      - title: Spring Boot
+        src: /assets/img/logos-technologies/logo-spring-boot.png
+      - title: Rails
+        src: /assets/img/logos-technologies/logo-rails.png
+      - title: Django
+        src: /assets/img/logos-technologies/logo-django.png
+
+  - heading: "Step 4: Run your app"
+    tab_block: site.data.enterprise-landing-page-tabs-images
+    image_side: left
+    body: |
+      Gruntwork Enterprise EKS supports various deployment strategies out of the box, including canary, blue/green
+      deployments, automatic rollbacks, security scans, CI/CD, GitOps workflows, etc. Post deployment, use powerful
+      observability tools to manage your apps, including dashboards for key metrics and cluster data, one-click
+      access to containers, and multiple options for centralized logging. Stay up to date automatically (see Patcher<!-- TODO: link to patcher page-->).
+    logos:
+      - title: Prometheus
+        src: /assets/img/logos-technologies/logo-prometheus.png
+      - title: Datadog
+        src: /assets/img/logos-technologies/logo-datadog.png
+
+  - heading: "Step 5: Get help and support"
+    image: /assets/img/landing-heroku-page/grunty-free-step-5.png
+    image_width: 250px
+    image_side: right
+    body: |
+      Enjoy consistent updates from Gruntwork, including updates to the latest releases of Kubernetes, Terraform, Docker, and other tooling. Our engineers can help when your team has questions.
+
+conclusion:
+  heading: The enterprise EKS experience you've been looking for.
```

Comment:
NIT: this section should be below the "meet the Gruntwork community" section. That's how we have it on all the other landing pages.

Reply:
Ben and I discussed this, and felt like the logos make it seems like the page ends there so we pulled up the CTA. We can revert it if the desire is to keep a consistent look and feel.